app/src/main/java/br/com/nespressohelper/daitan/nespressohelper/dbRead.java

```diff
@@ -0,0 +1,80 @@
+package br.com.nespressohelper.daitan.nespressohelper;
+
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+import java.util.ArrayList;
+
+/**
+ * Created by estrela on 12/21/17.
+ */
+
+public class dbRead {
+
+    public ArrayList<Coffee> getCoffees() {
+
+        SQLiteDatabase database = db.getInstance().getReadableDatabase();
+        String query = "SELECT * FROM " + db.TABLE_COFFEE;
+        ArrayList<Coffee> coffees = new ArrayList<>();
+
+        Cursor c = database.rawQuery(query, null);
+
+        if(c.moveToFirst()) {
+
+            do {
+                Coffee coffee = new Coffee();
+                coffee.setName(c.getString(0));
```

Comment:
Tip, you can use the `getColumnIndex` method to get those indexes by column name, this way if you add other columns you won't have to update the rest of the code.

Also, I find it useful to create an enum with the column names, or at least an static property. This way you can benefit from autocomplete when trying to remember a column name and your code has fewer hardcoded strings that need to be changed in bulk